src/Compat.jl

```diff
@@ -706,6 +706,42 @@ if VERSION < v"1.7.0-DEV.1187"
     export redirect_stdio
 end
 
+# https://github.com/JuliaLang/julia/pull/46104
+if VERSION < v"1.10.0-DEV.1404"
+    import Base: sort, Ordering, Forward, ord, lt, tail, copymutable, DEFAULT_STABLE, IteratorSize, HasShape, IsInfinite
```

Comment:
Shouldn't import things not meant to be extended:
```suggestion
    import Base: sort
    using Base: Ordering, Forward, ord, lt, tail, copymutable, DEFAULT_STABLE, IteratorSize, HasShape, IsInfinite
```
or (and that would be my personal preference)
```suggestion
    using Base: Ordering, Forward, ord, lt, tail, copymutable, DEFAULT_STABLE, IteratorSize, HasShape, IsInfinite
```
and explicitly qualify `sort` below.